src/EnergyPlus/AirflowNetwork/src/Elements.cpp

```diff
@@ -2059,6 +2905,8 @@ namespace AirflowNetwork {
     int DisSysCompHXProp::calculate(bool const LFLAG,           // Initialization flag.If = 1, use laminar relationship
                                     Real64 const PDROP,         // Total pressure drop across a component (P1 - P2) [Pa]
                                     int const EP_UNUSED(i),     // Linkage number
+                                    const Real64 EP_UNUSED(multiplier), // Element multiplier
+                                    const Real64 EP_UNUSED(control),    // Element control signal
```

Comment:
Adding unused arguments? I assume you'll need these later for some reason.

Reply:
Yes, eventually it'll be the link number that is unused, then it'll go away, and these will will get used in place of the link number argument.